deploy/docker/localkube-dind/Dockerfile

```diff
@@ -0,0 +1,71 @@
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM debian:jessie
+
+ARG CNI_RELEASE=0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff
```

Comment:
This is a bit of an old release?  

`14082201  2017-03-22T20:04:26Z  gs://kubernetes-release/network-plugins/cni-0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff.tar.gz#1490213066226000  metageneration=1`

Maybe use the github releases now?   https://github.com/containernetworking/cni/releases/download/$(CNI_BIN_VERSION)

Checkout deploy/iso/minikube-iso/package/cni-bin/Config.in